### PR TITLE
Use updated FAQs

### PR DESCRIPTION
**Pull request by Vinnl**
l10n dependency: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/44

### CODE REVIEW

File: privaterelay/templates/faq.html [hunk 1]
```diff
@@ -136,6 +137,10 @@ <h2 class="faq-headline">{% ftlmsg 'faq-question-unsubscribe-domain-question' %}
               <h2 class="faq-headline">{% ftlmsg 'faq-question-8-question' %}</h2>
               <p class="faq-answer js-set-href">{% ftlmsg 'faq-question-8-answer-html' url='https://www.mozilla.org/privacy/firefox-relay/' attrs='class="text-link" data-url="https://www.mozilla.org/privacy/firefox-relay/"' %}</p>
             </section>
+            <section class="faq" id="faq-data-collection">
```
Comment (maxxcrawford) [inline privaterelay/templates/faq.html:140]: Duplicated ID from preceding question 

```suggestion
            <section class="faq" id="faq-privacy-notice">
```

File: privaterelay/templates/faq.html [hunk 2]
```diff
@@ -136,6 +137,10 @@ <h2 class="faq-headline">{% ftlmsg 'faq-question-unsubscribe-domain-question' %}
...
+              <h2 class="faq-headline">{% ftlmsg 'faq-question-email-storage-question' %}</h2>
```
Comment (maxxcrawford) [inline privaterelay/templates/faq.html:141]: This question is only visible if premium is enabled. It needs to be in the above block too.

**Review by maxxcrawford — CHANGES_REQUESTED** [review]

**Review by Vinnl — COMMENTED** [review]
Geez rookie mistakes. I can't approve my own PR, but assuming the new question should indeed also be shown before Premium is enabled, your added commits look good to me :+1:

**Review by codemist — COMMENTED** [review]
Looks good, though I have to ask if 'When should I use Relay aliases' qn should disappear when `PREMIUM_ENABLED` is set to `False`, but remains visible on the landing page?

**Review by codemist — APPROVED** [review]

**Review by maxxcrawford — APPROVED** [review]
🤝

**Pull request merged**